wormhole-connect/src/routes/sdkv2/route.ts

```diff
@@ -45,6 +46,10 @@ export class SDKv2Route {
       this.NATIVE_GAS_DROPOFF_SUPPORTED = true;
       this.AUTOMATIC_DEPOSIT = true;
     }
+    this.IS_TOKEN_BRIDGE_ROUTE =
```

Comment:
please add a `TODO` above this to call out we should remove it some day 😭

Reply:
done